### PR TITLE
TLC-248

### PR DESCRIPTION
**Pull request by VitalyMikhailov**

### CODE REVIEW

File: pageObjects/register.page.js [hunk 1]
```diff
@@ -1,80 +1,61 @@
...
+  }
+
+  get registerBtn() {
+    return $('.ant-btn.ant-btn-primary.ant-btn-lg');
+  }
+
+  termsAndAgreements() {
+    this.agreementCheckBox.click();
+    this.termsAndAgreementsLink.click();
+    this.termsAndAgreementsLinkClose.waitForDisplayed({ timeout: 10000 });
```
Comment (jstepiko) [inline pageObjects/register.page.js:44]: It would better no to hardcode timeouts

File: pageObjects/register.page.js [hunk 2]
```diff
@@ -1,80 +1,61 @@
...
+    return $('#user_login_agreement');
+  }
+
+  get registerBtn() {
+    return $('.ant-btn.ant-btn-primary.ant-btn-lg');
+  }
+
+  termsAndAgreements() {
+    this.agreementCheckBox.click();
+    this.termsAndAgreementsLink.click();
```
Comment (gennadyGOR) [inline pageObjects/register.page.js:43]: cause problems on MAC OS

File: pageObjects/register.page.js [hunk 3]
```diff
@@ -1,80 +1,61 @@
...
+  get email() {
+    return $('#user_login_email');
+  }
+
+  get password() {
+    return $('#user_login_password');
+  }
+
+  get termsAndAgreementsLink() {
+    return $('.ant-btn.pl-1.ant-btn-link.ant-btn-sm');
```
Comment (vasylynash) [inline pageObjects/register.page.js:26]: Optimize selectors. E.g. $$("button")[0]

File: pageObjects/register.page.js [hunk 4]
```diff
@@ -1,80 +1,61 @@
...
+  get termsAndAgreementsLinkClose() {
+    return $('[data-icon="close"]');
+  }
+
+  get agreementCheckBox() {
+    return $('#user_login_agreement');
+  }
+
+  get registerBtn() {
+    return $('.ant-btn.ant-btn-primary.ant-btn-lg');
```
Comment (vasylynash) [inline pageObjects/register.page.js:38]: Optimize selectors. E.g.$$("button")[1]

**Pull request merged**